build.gradle

```diff
@@ -358,7 +358,9 @@ task reference(type: org.asciidoctor.gradle.AsciidoctorTask) {
 		}
 	options doctype: 'book', eruby: 'erubis'
 	attributes 'source-highlighter': 'coderay',
-		'icons': 'font'
+		'icons': 'font',
+		'build-gradle': file('build.gradle'),
+		'spring-amqp-version': "${project.version}"
```

Comment:
Why can't we just live with an existing `project-version` property?
Why do we need to introduce a new one for the same meaning?

We are here really in the scope of the current project and don't build any composite docs with any other projects...